src/components/pages/AdminAddCourse/AdminAddCoursesForm.js

```diff
@@ -96,18 +102,44 @@ function AdminAddCoursesForm(props) {
     setIsModalVisible(false);
   };
 
+  const { dispatch } = useDispatch;
```

Comment:
I believe you need to have parentheses following this hook?

Reply:
Brilliant Max, nice catch!